src/library/scala/ref/PhantomReference.scala

```diff
@@ -15,6 +15,7 @@ package scala.ref
 /**
  *  @author Sean McDirmid
  */
+@deprecated("Use `java.lang.ref.PhantomReference` instead.",since = "2.13.0-RC1")
```

Comment:
probably `since = 2.13.0`, as deprecation in an RC is not meaningful. Also missing space after comma.

Reply:
Thanks